src/Analysis/Engine/Test/TypeAnnotationTests.cs

```diff
@@ -392,13 +391,13 @@ private class StringConverter : TypeAnnotationConverter<string> {
 new_vector = scale(2.0, [1.0, -4.2, 5.4])
 ",
                 new[] {
-                    "scale:scale(scalar:float=,vector:list[float]=)->[list,list[float]]"
+                    "scale:scale(scalar:float, vector:list[float]) -> list[float]"
```

Comment:
So `[list,list[float]]` is actually merged into `list[float]`?

Reply:
Yes. Tooltips were correct while signatures were really weird